README.md

```diff
@@ -103,7 +103,7 @@ Any Java code that is executed in the context of a process is usually written in
 1. Run the application `mvn clean spring-boot:run`
 2. Deploy all the processes by running scripts/deploy.sh file
 3. Navigate to Camunda Portal `localhost:9000/app/welcome/default/#/welcome`
-4. Log in as admin username: `admin`, password: `admin`
+4. Log in as admin user: `admin`, password: `admin`
```

Comment:
I think this is missing punctuation on not incorrect usage of `username`.

Shouldn't it be like:

```
4. Log in as admin. username: `admin`, password: `admin`
```